release/.buildkite/build_pipeline.py

```diff
@@ -61,7 +61,9 @@ def __init__(self, name: str, retry: int = 0):
         "shuffle_1tb_1000_partition",
         "non_streaming_shuffle_1tb_1000_partition",
         "shuffle_1tb_5000_partitions",
-        "non_streaming_shuffle_1tb_5000_partitions",
+        # TODO(sang): It doesn't even work without spilling
```

Comment:
maybe create an issue/or add a bit details on the limit?

Reply:
Sure. I can do that.

Reply:
https://github.com/ray-project/ray/issues/18333